src/witan/app/forecast.clj

```diff
@@ -116,13 +126,70 @@
                             :owner owner
                             :in_progress false}))
 
+
+
+(defn add-to-result-values
+  [result name value]
+  (assoc result :values (assoc (:values result) name (hayt/user-type {:name name :value value}))))
```

Comment:
It looks like you are updating the value stored in :values key of result?

If that's true, `update-in` seems better?

`(update-in result [:values] assoc name (hayt/user-type {:name name :value value}))`